lib/postgres.js

```diff
@@ -288,11 +310,11 @@ class Postgres {
 
         // Builds binding pattern such as ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11::jsonb, now(), $12, NULL, $13)
         for (let c = 0; c < itemCount; c++) {
-            values.push(`($${i++}, $${i++}, $${i++}, $${i++}, $${i++}, $${i++}, $${i++}, $${i++}, $${i++}, $${i++}, $${i++}::jsonb, now(), $${i++}, NULL, $${i++})`);
+            values.push(`($${i++}, $${i++}, $${i++}, $${i++}, $${i++}, $${i++}, $${i++}, $${i++}, $${i++}, $${i++}, $${i++}::jsonb, now(), $${i++}, NULL, $${i++}, $${i++}::jsonb)`);
         }
 
-        return `INSERT INTO items (ms, a, d, paintseed, paintwear, defindex, paintindex, stattrak, souvenir, props, stickers, updated, rarity, floatid, price)
-                VALUES ${values.join(', ')} ON CONFLICT(defindex, paintindex, paintwear, paintseed) DO UPDATE SET ms=excluded.ms, a=excluded.a, d=excluded.d, stickers=excluded.stickers, updated=now()`;
+        return `INSERT INTO items (ms, a, d, paintseed, paintwear, defindex, paintindex, stattrak, souvenir, props, stickers, updated, rarity, floatid, price, keychains)
```

Comment:
nit: Move the `keychains` value next to the `stickers` value when updating.

Reply:
Thank you very much for reviewing my PR!  I’ll address the feedback and plan to submit the changes tomorrow.

For this PR, I will:

- Save all fields for `keychains` and adjust the order of the `keychains` field.
- Remove the logic: `&& item.defindex !== 1355`.

I’ll also create a new branch to add an optional config for saving charms data to the cache.

Thanks again for your help!